assets/layers/recycling/recycling.json

```diff
@@ -0,0 +1,532 @@
+        },
+        {
+          "if": "recycling:plastic=yes",
+          "then": "circle:#FFFFFF;./assets/layers/recycling/plastic.svg"
+        },
+        {
+          "if": "recycling:scrap_metal=yes",
+          "then": "circle:#FFFFFF;./assets/layers/recycling/scrap_metal.svg"
+        },
+        {
+          "if": "recyling:shoes=yes",
+          "then": "circle:#FFFFFF;./assets/layers/recycling/shoes.svg"
+        },
+        {
+          "if": "recycling:small_electrical_appliances=yes",
+          "then": "circle:#FFFFFF;./assets/layers/recycling/small_electrical_appliances.svg"
+        },
+        {
+          "if": "recycling:waste=yes",
+          "then": "circle:#FFFFFF;./assets/layers/waste_disposal/waste_disposal.svg"
+        }
+      ]
+    }
+  ],
+  "presets": [
+    {
+      "title": {
+        "en": "recycling container",
+        "nl": "recycling container"
+      },
+      "tags": [
+        "amenity=recycling",
+        "recycling_type=container"
+      ]
+    },
+    {
+      "title": {
+        "en": "recycling centre",
+        "nl": "recycling centre"
+      },
+      "tags": [
+        "amenity=recycling",
+        "recycling_type=centre"
+      ]
+    }
+  ],
+  "tagRenderings": [
+    {
+      "id": "recycling-type",
+      "question": {
+        "en": "What type of recycling is this?",
+        "nl": "Wat voor soort recycling is dit?"
+      },
+      "mappings": [
+        {
+          "if": "recycling_type=container",
+          "then": {
+            "en": "This is a recycling container",
+            "nl": "Dit is een recyclingcontainer"
+          }
+        },
+        {
+          "if": "recycling_type=centre",
+          "then": {
+            "en": "This is a recycling centre",
+            "nl": "Dit is een recyclingcentrum"
+          }
+        }
+      ]
+    },
+    {
+      "id": "container-location",
+      "question": {
+        "en": "Where is this container located?",
+        "nl": "Waar bevindt deze container zich?"
+      },
+      "condition": "recycling_type=container",
+      "mappings": [
+        {
+          "if": "location=underground",
+          "then": {
+            "en": "This is an underground container",
+            "nl": "Dit is een ondergrondse container"
+          }
+        },
+        {
+          "if": "location=indoor",
+          "then": {
+            "en": "This container is located indoors",
+            "nl": "Deze container bevindt zich binnen"
+          }
+        },
+        {
+          "if": "location=",
+          "then": {
+            "en": "This container is located outdoors",
+            "nl": "Deze container is buiten"
+          }
+        }
+      ]
+    },
+    {
+      "id": "recycling-accepts",
+      "question": {
+        "en": "What can be recycled here?",
+        "nl": "Wat kan hier gerecycled worden?"
+      },
+      "multiAnswer": true,
+      "mappings": [
```

Comment:
I'm missing 'organic waste' a bit here, that would be a useful addition.

Reply:
Currently it does include `recycling:green_waste`, which feels very similar, and seems to be used more